start.py

```diff
@@ -55,7 +56,17 @@ def main(simulation_parameters):
             allagents.do('filobl')
             insurancecustomers.do('check_risk')
 
-        simulation.graphs()
+        if not isTravis:
+            simulation.graphs()
```

Comment:
Here is the line where the graphs are automatically produced and the web server launched. I'd recommend to having the data analysis residing in a Jupyter notebook actually. This offloads the graphs development effort to another library.